frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -239,6 +240,11 @@ export function DynamicConfiguration () {
                         text={state.form.configuration.SystemBanner.text}
                         backgroundColor={state.form.configuration.SystemBanner.backgroundColor}
                         textColor={state.form.configuration.SystemBanner.textColor} />
+                    { !window.env.MANAGE_IAM_ROLES && 
+                        <Alert statusIconAriaLabel='Info'>
+                            {window.env.APPLICATION_NAME} has the ability to configure which services are available and what instance types are available if dynamic roles are enabled
+                        </Alert>
+                    }
```

Comment:
We should try and get wording from Cindy if we want to promote the usage of dynamic roles I think it could be a warning and make it obvious from the start. Maybe something like this?

![image](https://github.com/awslabs/mlspace/assets/14645/62c08f3a-6404-464e-9f0d-be9f9e4d2cea)

Reply:
Probably makes the most sense to have this at the top of the config page vs the bottom too, that's generally where these types of messages appear.

Reply:
I can move it. I will consult Cindy from a user perspective. My thought was that I wouldn't constantly have to have a semi non-relevant box displayed, but maybe that is the point.

Reply:
>a little too judgemental 

but that's the point 😉

Top of the page has my vote as well - this admin page is something that will be used very intermittently. This way when they use the page again after not touching it for a few weeks/months and can't remember why 2 configurations aren't visible (which are in documentation) they'll know why.